api/register/swagger-api_register.yaml

```diff
@@ -73,11 +92,11 @@ paths:
           description: Invalid information provided
         403:
           description: Invalid RECAPTCHA token
-  /register/config:
+  /register/realms/{realm}/user/profile:
     get:
       tags:
       - Configuration
-      summary: Gets configuration
+      summary: Gets the definition of user attributes from the realm specified in URL path
       parameters:
       - name: realm
         in: query
```

Comment:
path not query

Reply:
> path not query

Copy/paste of errors present in the repo for 3 years...